lib/computer_move.ex

```diff
@@ -1,12 +1,14 @@
+import WinningCombo
+
 defmodule FirstComputerSpot do
 
   def takeFirstAvailableSpot(board, token) do
```

Comment:
Will `takeFirstAvailableSpot` ever get called with a board that has a finished game? If not, then I think it would be nice to remove the dependency on IO here. Could maybe just raise an error instead.

Reply:
@davetorre I think it does check for a win in case the user before it took a winning spot. If this seems inappropriate and should bee in the higher level function let me know.

Reply:
Meeting Note: Take IO out of computer move for single responsibility

Reply:
IO was not able to be completely removed because replace_first_spot uses it and the end of the recursive function for Cat's Game. However, IO and dead code was removed from take_first_available_spot. This decoupled the WinningCombo dependency from the function and module.